browse/templates/abs/labs_tabs.html

```diff
@@ -80,6 +80,22 @@ <h1>Bibliographic and Citation Tools</h1>
     <div class="tab">
       <h1>Code, Data and Media Associated with this Article</h1>
       <div class="toggle">
+        <div class="columns is-mobile lab-row">
+          <div class="column lab-switch">
+            <label class="switch">
+              <input        
+                id="catalyzex-toggle"
+                data-script-url="{{ url_for('static', filename='js/catalyzex.js') }}"
+                type="checkbox" class="lab-toggle" aria-labelledby="label-for-cx">
+              <span class="slider"></span>
+              <span class="is-sr-only">Links to Code Toggle</span>
+            </label>
+          </div>
+          <div class="column lab-name">
+            <span id="label-for-cx">Code implementations on CatalyzeX</span> <em>(<a href="https://www.catalyzex.com" target="_blank">What is CatalyzeX?</a>)</em>
```

Comment:
```suggestion
            <span id="label-for-cx">CatalyzeX code for papers</span> <em>(<a href="https://www.catalyzex.com" target="_blank">What is CatalyzeX?</a>)</em>
```

Reply:
@gragtah  What's the reason for this copy/text change? I like 'Code implementations on CatalyzeX' more. It's easier to process. 

"CatalyzeX code ..." sounds like something else completely